cmd/fleetctl/preview.go

```diff
@@ -411,6 +424,10 @@ func unzip(r *zip.Reader, branch string) error {
 	}
 
 	for _, f := range r.File {
+		// Prevent zip-slip attack.
+		if strings.Contains(f.Name, "..") {
+			return fmt.Errorf("invalid path in zip: %q", f.Name)
+		}
```

Comment:
Added while I was at it, to fix https://github.com/fleetdm/fleet/security/code-scanning/15 and move forward in having cleaner code scanning alerts.

Reply:
Nice, thank you! Easy enough to do the check even though it wasn't an issue in our case because we make the zip files.